deno_typescript/compiler_main.js

```diff
@@ -279,6 +266,17 @@ class Host {
     /** @type {ts.ResolvedModule[]} */
     const r = resolvedNames.map(resolvedFileName => {
       const extension = getExtension(resolvedFileName);
+      if (!moduleMap.has(resolvedFileName)) {
+        // If we match the external specifier regex, we will then create an internal
+        // specifier and then use that when creating the source file
+        const result = externalSpecifierRegEx.exec(resolvedFileName);
```

Comment:
In the comment it would be nice to have an a real-world example of what sort of external specifies are being matched here. I don't follow this...

Reply:
Ok...  But `file://foo/bar/baz/deno/js/compiler.ts` gets transformed into `$deno$/js/compiler.ts`.  I have simply moved what was already there.

Reply:
My bad. Ignore.